README.md

```diff
@@ -70,7 +70,7 @@ rpm -i https://github.com/fortio/fortio/releases/download/v1.65.0/fortio-1.65.0-
 # and more, see assets in release page
 ```
 
-On a MacOS you can also install Fortio using [Homebrew](https://brew.sh/):
+On a macOS you can also install fortio using [Homebrew](https://brew.sh/):
```

Comment:
this probably should be
```
On a Mac ...
```
or
```
On Mac OS ...
```
But I guess you're technically correct officially it's macOS (or ... Darwin I guess but nobody knows that term)

What's wrong with `Fortio` btw

Reply:
it's no longer Mac OS, but macOS

https://en.wikipedia.org/wiki/MacOS

> macOS, originally Mac OS X, previously shortened as OS X

Side note, I'm not an Apple-fanboy :grin:

Reply:
about Fortio I hesitated, your documentation and code comments use randomly Fortio or fortio, I found fortio more often, so I changed it here, unsure why, maybe because I was updating the line

Reply:
It's Fortio (Φορτίο even) when talking about the project but the cli is `fortio`

so here it's (I think) Fortio even if it's to end up with `fortio` binary